tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/Core/InstrumentedTool.cs

```diff
@@ -46,6 +46,24 @@ public override async ValueTask<CallToolResult> InvokeAsync(RequestContext<CallT
             activity?.SetTag(TagName.ToolResponse, content);
             activity?.SetStatus(ActivityStatusCode.Ok);
 
+            if (result.Content?.FirstOrDefault() is TextContentBlock contentBlock)
+            {
```

Comment:
The `if` condition here is kind of odd to me. Why doesn't InvokeAsync only return (result.Content[0] as TextContentBlock) value, if we're going to throw away all the other content?

What is the other content in this case? Is it not valuable for telemetry?